apps/web/src/views/Swap/index.tsx

```diff
@@ -69,10 +69,10 @@ export default function Swap() {
 
   // Check if pancakeswap route is better than akka route or not
   useEffect(() => {
-    if (akkaRouterTrade?.route?.returnAmount && v2Trade?.outputAmount?.toSignificant(6)) {
-      if (Number(v2Trade?.outputAmount?.toSignificant(6)) > Number(akkaRouterTrade?.route?.returnAmount)) {
+    if (akkaRouterTrade?.route?.returnAmountWei && v2Trade?.outputAmount) {
+      if (v2Trade?.outputAmount.numerator.toString() > akkaRouterTrade?.route?.returnAmountWei) {
```

Comment:
looks like a comparision between string and number

Reply:
> looks like a comparision between string and number

It's string now but we are going to compare both using bigNumber